app/src/main/res/drawable/general_item_background_border_grey.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <solid android:color="@color/component_color_profile_rename_activity_secondary_background_color" />
+  <stroke
+    android:width="1dp"
+    android:color="@color/color_def_black_12" />
+</shape>
```

Comment:
EOF missing.

Reply:
done @Sarthak2601

Reply:
still pending. 
add a blank line at the end.

Reply:
It's still pending but I think @rt4914 has also commented on the same line, so I'll resolve this conversation for now.